documentation/develop/firefox-workflow-overview.md

```diff
@@ -52,33 +52,33 @@ date: 2019-05-20 03:25:40
         <td>Choose your IDE or code editor</td>
         <td>Run your extension with <a href="/documentation/develop/getting-started-with-web-ext#testing-out-an-extension">web-ext run</a> or <a href="/documentation/develop/temporary-installation-in-firefox/">about:debugging</a></td>
         <td>Create an <a href="https://addons.mozilla.org">addons.mozilla.org</a>account</td>
-        <td>Promote your extension</td>
+      <td><a href="/documentation/publish/promoting-your-extension/"Promote your extension</a></td>
```

Comment:
Ah, it looks like the problem here is that you need a `>` at the end of <a href="/documentation/publish/promoting-your-extension/".

Reply:
Fixed this! 😸